api/src/reportcreator_api/pentests/models/common.py

```diff
@@ -128,6 +128,7 @@ class Language(models.TextChoices):
     ENGLISH_CA = 'en-CA', True, 'English (en-CA)'
     ENGLISH_NZ = 'en-NZ', True, 'English (en-NZ)'
     ENGLISH_ZA = 'en-ZA', True, 'English (en-ZA)'
+    CZECH = 'cs-CZ', True, 'Czech (cs-CZ)'
```

Comment:
Languagetool does not support spellchecking Czech (list of supported languages: https://dev.languagetool.org/languages). Please set the `spellcheck` argument (second item in tuple) to `False` and move the `CZECH` entry into the section below ("Languages without LanguageTool support").

Reply:
Unfortunately the app fails with "root: 'cs-CZ' is not a valid Language" while opening the login page. I've added the language code to preferred languages:

PREFERRED_LANGUAGES="cs-CZ,de-DE,en-US,sk-SK"